src/checkpoint/fevm/topdown.rs

```diff
@@ -99,22 +99,11 @@ impl<T: EthManager + Send + Sync> CheckpointManager for TopdownCheckpointManager
 
     async fn submit_checkpoint(
         &self,
-        epoch: ChainEpoch,
+        _epoch: ChainEpoch,
         // TODO: when we support more wallet addresses, we need this variable
         _validator: &Address,
     ) -> anyhow::Result<()> {
-        let msgs = self
-            .parent_manager
-            .top_down_msgs(&self.child_subnet.id, epoch)
-            .await?;
-        let checkpoint = gateway::TopDownCheckpoint {
-            epoch: epoch as u64,
-            top_down_msgs: msgs,
-        };
-        self.child_manager
-            .submit_top_down_checkpoint(checkpoint)
-            .await?;
-        Ok(())
+        todo!()
```

Comment:
Why do we remove this one? This is the one that handles fevm-fevm, right?

Reply:
Because there are quite some changes due to API update, might need some rewrite, `todo` is easier for now.